frontend/micro-ui/web/micro-ui-internals/packages/modules/pt/src/pages/employee/NewApplication/PropertyDetailsTableSection.js

```
@@ -0,0 +1,564 @@
+
+  const floors = FloorAll?.PropertyTax?.Floor || [];
+
+  const mappedFloors = floors
+    .filter(floor => floor?.code && floor?.active)
+    .map(floor => ({
+      i18nKey: floor.name,
+      code: floor.code,
+    }))
+    .sort((a, b) => {
+      const getSortValue = (val) => {
+        const num = parseInt(val, 10);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)